src/src/Controller/AboutController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+class AboutController extends AbstractController
+{
+    #[Route('/about', name: 'app_about')]
+    public function index(): Response
+    {
+        return $this->render('about/index.html.twig', [
+            'controller_name' => 'AboutController',
```

Comment:
This variable is generated by the framework as demo code. It should be removed as you don't need it.